internal/metrics/metrics.go

```diff
@@ -0,0 +1,114 @@
+package metrics
+
+
+import (
+	"time"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/gin-gonic/gin"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+const (
+	ms = 0.001
+	s  = 1000 * ms
```

Comment:
Do I understand correctly that this thing works because s =1 and you give data in seconds? If so, maybe this should be explicitly defined as 1?